### PR TITLE
Fixed size and color of buttons and text boxes

### PR DESCRIPTION
**Pull request by monika-Dianomic**

### CODE REVIEW

File: src/app/components/core/scheduler/list-schedules/list-schedules.component.html [hunk 1]
```diff
@@ -35,6 +35,6 @@
     </table>
   </div>
 </div>
-<app-alert-dialog (enable)='enableSchedule($event)' (disable)='disableSchedule($event)' (delete)='deleteSchedule($event)'
-  [childData]='childData'></app-alert-dialog>
+<!-- <app-alert-dialog (enable)='enableSchedule($event)' (disable)='disableSchedule($event)' (delete)='deleteSchedule($event)'
```
Comment (praveen-garg) [inline src/app/components/core/scheduler/list-schedules/list-schedules.component.html]: why this has been commented?

File: src/app/components/common/alert-dialog/alert-dialog.component.ts [hunk 2]
```diff
@@ -23,6 +23,24 @@ export class AlertDialogComponent implements OnInit {
...
+      this.childData.action = 'Delete';
+    }
+    if (this.childData.key === 'logout' || this.childData.key === 'clearSessions') {
+      this.childData.action = 'Logout';
+    }
+    if (this.childData.key === 'clearSessions') {
+      this.childData.action = 'Clear Sessions';
+    }
+    if (this.childData.key === 'createBackup') {
+      this.childData.action = 'Create';
```
Comment (praveen-garg) [inline src/app/components/common/alert-dialog/alert-dialog.component.ts]: ACTION dispatcher should be in SCREAMING_SNAKE_CASE e.g. CLEAR_SESSION

Comment (monika-Dianomic) [reply src/app/components/common/alert-dialog/alert-dialog.component.ts]: This action is to be displayed as a `text on button` in alert dialog; just like:

alert-dialog

So can not be in SNAKE_CASE

Comment (praveen-garg) [reply src/app/components/common/alert-dialog/alert-dialog.component.ts]: you mean this not action itself, but actionButtonValue ?


BTW, in this screenshot there is one extra space before ` user`, it should not be

File: src/app/components/common/alert-dialog/alert-dialog.component.ts [hunk 3]
```diff
@@ -23,6 +23,24 @@ export class AlertDialogComponent implements OnInit {
...
 
+  ngOnChanges() {
+    if (this.childData.key === 'restoreBackup') {
+      this.childData.action = 'Restore';
+    }
+    if (this.childData.key === 'deleteBackup' || this.childData.key === 'deleteCertificate' || this.childData.key === 'deleteUser') {
+      this.childData.action = 'Delete';
+    }
+    if (this.childData.key === 'logout' || this.childData.key === 'clearSessions') {
+      this.childData.action = 'Logout';
```
Comment (praveen-garg) [inline src/app/components/common/alert-dialog/alert-dialog.component.ts]: `Log Out`

**Pull request merged**